spec/index.bs

```diff
@@ -9628,11 +9628,12 @@ dictionary GPUCommandEncoderDescriptor
                     <div class=validusage>
                         - |descriptor| must meet the [$GPURenderPassDescriptor/Valid Usage$] rules
                             given device |this|.{{GPUObjectBase/[[device]]}}.
-                        - The set of attachments in |descriptor|.{{GPURenderPassDescriptor/colorAttachments}}
+                        - The set of texture regions in |descriptor|.{{GPURenderPassDescriptor/colorAttachments}}
                             must be pairwise disjoint.
-                            That is, no two attachments may refer to the same region, as defined by
-                            the {{GPURenderPassColorAttachment/view}}'s [=texture subresource=]
-                            range and (for {{GPUTextureViewDimension/"3d"}} attachments)
+                            That is, no two texture regions may overlap, where
+                            each attachment has 1-2 regions defined by
+                            the {{GPURenderPassColorAttachment/view}}'s and {{GPURenderPassColorAttachment/resolveTarget}}'s
+                            [=texture subresource=] ranges, and (for {{GPUTextureViewDimension/"3d"}} attachments)
                             the attachment's {{GPURenderPassColorAttachment/depthSlice}}.
                     </div>
                 1. Consider each [=texture subresource=] viewed by a non-`null` element of
```

Comment:
While looking at the other issue I realized this is pretty ambiguous too. Should update this to clarify that "viewed by" means both `view` and `resolveTarget`.